src/nvim/option.c

```diff
@@ -1215,16 +1215,10 @@ do_set (
                 goto skip;
               }
             }
```

Comment:
You need to move this on the same line as `else`. New linting policy requires fixing lint errors in lines adjacent to the changed ones or it will not pass tests.